app/Http/Controllers/FeedbackController.php

```diff
@@ -319,7 +323,7 @@ public function categoryUpdate(Request $request, int $id)
         $category->show_publisher = $request->has('show_publisher');
         $category->save();
 
-        Session::flash('flash_message', 'The category '.$category->name.' has been updated.');
```

Comment:
isnt this a different field?

Reply:
Yeah it was wrong before, it should be title